src/services/item/plugins/app/fluent-schema.ts

```diff
@@ -8,12 +8,23 @@ export const appSchema = S.object()
   .prop('url', S.string())
   .prop('extra', S.object().additionalProperties(true));
 
+export const mostUsedAppSchema = S.object()
+  .additionalProperties(false)
+  .prop('name', S.string())
+  .prop('url', S.string())
+  .prop('nbr', S.number());
```

Comment:
As I suggested in builder:
```suggestion
  .prop('count', S.number());
```